.github/workflows/test-runner-on-perf.yml

```diff
@@ -94,3 +94,14 @@ jobs:
           gcloud container clusters update ${K8_CLUSTER} \
             --enable-master-authorized-networks \
             --master-authorized-networks ${RESTORE_IPS}
+      - name: "Notify Slack"
+        if: always()
+        uses: broadinstitute/action-slack@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        with:
+          status: ${{ job.status }}
+          channel: "#dsde-qa"
```

Comment:
would there be any benefit to pinging our internal channel as well?

Reply:
Good idea.